web/concrete/elements/page_controls_footer.php

```diff
@@ -640,6 +685,27 @@ class="btn btn-xs <?= $act->getWorkflowProgressActionStyleClass() ?>"><?= $act->
                             ))?>
 
                 <?php
+
+                }
+            } else {
+                $publishDate = $vo->getPublishDate();
+                if ($publishDate) {
+                    $date = $dateHelper->formatDate($publishDate);
+                    $time = $dateHelper->formatTime($publishDate);
+                    $message = t("This version of the page is scheduled to be published on " . $date .
```

Comment:
I think you should use the t function like this: t("This ...to be on %s...at %s", $var1, $var2);